app/services/fetch.js

```diff
@@ -13,4 +13,16 @@ export default fetch.extend({
 
     this._super(...arguments);
   },
+
+  fetchFromMediaWikiAuthenticated(requestUrl, accessToken, errorClass) {
```

Comment:
This could be put in ember-fandom but I'm fine with giving I-Wing the chance to do it :wink:

Reply:
Added a TODO in 633bcb17b5ca85bfcc67173eb4194d663df23a86.